src/js/feedback/components/FeedbackForm.jsx

```diff
@@ -41,9 +42,18 @@ class FeedbackForm extends React.Component {
     this.props.setFormValues({ email });
   }
 
-  sendFeedback = (event) => {
+  setResponse = ({ value: shouldSendResponse, dirty }) => {
+    //console.log('value: ', value, 'dirty:', dirty)
+    this.setState({ suppressResponseErrors: true })
+    this.props.setFormValues({ shouldSendResponse });
+  }
+
+  onSubmit = (event) => {
     event.preventDefault();
-    if (this.props.formIsSubmittable) this.props.sendFeedback(this.props.formValues);
+    if (this.props.formValues.shouldSendResponse) return this.props.sendFeedback();
+    //Set the redux error message
```

Comment:
It looks like we can reuse this.setResponse here:
```
-    this.props.setFormValues({shouldSendResponse: null});
-    this.setState({ suppressResponseErrors: false });
+   this.setResponse({{value: null});
```
Then update setResponse:
```
+  setResponse = ({ value: shouldSendResponse}) => {
+    this.setState({ suppressResponseErrors: false })
+    this.props.setFormValues({ shouldSendResponse });
+  }
```

Reply:
Doing this introduced a fun new error, but it works great otherwise, so I'm going to leave it as-is for the moment.